### ISSUE TITLE
RangeError: Maximum call stack size exceeded

### ISSUE THREAD

**Issue by earnubs** [mozilla/npm-lockdown#40]
```
$ node_modules/.bin/lockdown-relock
module.js:160
  var trailingSlash = (request.slice(-1) === '/');
                                         ^
RangeError: Maximum call stack size exceeded
    at Object.stringify (native)
    at Function.Module._findPath (module.js:162:23)
    at Function.Module._resolveFilename (module.js:334:25)
    at Function.Module._load (module.js:278:25)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at getShasum (/Users/sstewart/Workspace/webdm/workspace/node_modules/lockdown/relock.js:100:15)
    at walk (/Users/sstewart/Workspace/webdm/workspace/node_modules/lockdown/relock.js:71:14)
    at /Users/sstewart/Workspace/webdm/workspace/node_modules/lockdown/relock.js:80:7
    at Array.forEach (native)
```

**Comment by pdehaan** [comment]
Possibly related to #31

**Comment by indrek-rajamets** [comment]
Anyone working with this issue?

**Comment by vladikoff** [comment]
@indrek-rajamets should be fixed in 0.0.8
